style/ember/README.md

```diff
@@ -2,6 +2,7 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Don't use {{partial}} helper. Use a component with only a template.
```

Comment:
I don't think it will be deprecated. I think partial serves exactly the purpose I reach for it which is "this is a large section of markup that I'd like to organize. I will not be using it elsewhere I just want to make this template cleaner by removing some stuff from view".